builtin/common/chatcommands.lua

```diff
@@ -65,9 +56,8 @@ local function do_help_cmd(name, param)
 			end
 		end
 		table.sort(cmds)
-		return true, gettext("Available commands: ") .. table.concat(cmds, " ") .. "\n"
-				.. gettext_replace("Use '$1help <cmd>' to get more information,"
-				.. " or '$1help all' to list everything.", cmd_marker)
+		return true, S("Available commands: @1", table.concat(cmds, " ")) .. "\n"
```

Comment:
```suggestion
		return true, S("Available commands:") .. " " .. table.concat(cmds, " ") .. "\n"
```
to avoid duplicate translations